documentation/concepts/!concept-template.md

```diff
@@ -0,0 +1,103 @@
+---
+title: {Concept Name}
+---
+<!-- template preamble -->
+
+## Template preamble (remove!)
+
+A concept page explains some aspect of the API Management Platform. They can be
+thought of as extended definitions of major abstractions, such as a resource
+(like Products) or higher-level concept (like an API Gateway) or an overview of
+the Platform. Concept pages provide essential background, enabling readers to
+understand the Platform, and thereby navigate tasks to achieve their goals.
+Typically, concept pages don't include sequences of steps, but instead provide
+links to tasks or tutorials.
+
+Considerations for page audience:
+
+- Layer depths of explanation: Start with simple, high-level explanations for
+  non-technical readers, then progress into detailed technical explanations.
+- If that isn't enough, then consider dividing the concept page into sections
+  for different audiences.
+
+To write a new concept page, copy this file. All text in {curly brackets} should
+be replaced or removed.
+
+For more information on concept pages, visit the [Good Docs Project](https://gitlab.com/tgdp/templates/-/tree/main/concept).
+
+---
+
+<!-- overview -->
+
+{A summary paragraph introducing a concept, explaining its importance or
+relevance, and providing an overview of the content that will be covered
+in the document (scope).}
+
+{Typical wording to use is:}
+
+This article explains the basics of {concept} and how it works in {the tool or context}.
+
+{Then include a paragraph with a definition of the concept you are explaining.
+If more definitions are needed, include those definitions here as a bulleted list.}
+
+{Optional: Add visual aids to complement explanations (system diagram, flowchart, decision tree) -
+see the [Style Guide - Diagrams](/reference/style-guide.md#diagrams) and
+[Good Docs](https://gitlab.com/tgdp/templates/-/blob/main/concept/process-concept.md#create-visual-aids-for-a-concept-document).}
+
+(Optional) Image/Figure: {Image title, which concisely explains the image or
+figure.}
+
+<!-- body -->
+
+## Background (optional)
+
+{Use this section to provide a reader with a context, prehistory, or background information.}
+
+Typical wordings to use are:
+
+- The reason {X} is designed that way is because historically, ...
+- The idea of {X} originated from the growing demand for ...
+
+## Use cases OR Purpose
+
+{Answer "How can I use it?" or "How does it help me?" from the reader's perspective.
+Use this section to explain the overall purpose and provide use
+cases to show how a reader can benefit from a concept.}
+
+## Comparison of {thing being compared} (optional)
+
+{Use this section to compare options or alternatives within a concept.}
+
+Table: {Table title which concisely explains the comparison.}
+
+| Option   | Pros                                  | Cons                                  |
```

Comment:
I don't think it's necessarily always pros/cons may want to remove this from the template to give more room to compare various attributes not necessarily pro/con

Reply:
100% agree there could be other comparisons to make (e.g. different use cases). I thought this would be clear enough to writers from the section heading and preceding content. If you disagree we can add a note to say as much.

I'd like to keep the table as tables are very useful for comparisons and we don't really do that much (at all?) yet.

Reply:
I like the table but may want to adjust the headings or indicate that pro/con is an example @rustyjux

Reply:
Hold the phones - there is another table below which was already there as an alternative to this pros/cons table. Marking as resolved, reopen if need be.